tests/helper.py

```diff
@@ -4,12 +4,105 @@
 import os
 import random
 import string
+import sys
 import unittest2
 
-from mock import patch, Mock
+from distutils.version import StrictVersion
+from mock import patch, Mock, ANY
 
 import stripe
 from stripe import six
+from stripe.six.moves.urllib.request import urlopen
+from stripe.six.moves.urllib.error import HTTPError
+
+
+MOCK_MINIMUM_VERSION = '0.4.0'
+MOCK_PORT = os.environ.get('STRIPE_MOCK_PORT', 12111)
+
+
+try:
+    resp = urlopen('http://localhost:%s/' % MOCK_PORT)
+    info = resp.info()
+except HTTPError as e:
+    info = e.info()
+except Exception:
+    sys.exit("Couldn't reach stripe-mock at `localhost:%s`. Is "
+             "it running? Please see README for setup instructions." %
+             MOCK_PORT)
+
+version = info.get('Stripe-Mock-Version')
+if version != 'master' \
+        and StrictVersion(version) < StrictVersion(MOCK_MINIMUM_VERSION):
+    sys.exit("Your version of stripe-mock (%s) is too old. The minimum "
+             "version to run this test suite is %s. Please "
+             "see its repository for upgrade instructions." %
+             (version, MOCK_MINIMUM_VERSION))
```

Comment:
Thanks for copying these various checks over! I think they're fairly ergonomically advantageous for people trying to make use of stripe-mock, so it's important to port them everywhere that we port stripe-mock.